src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -170,6 +163,7 @@ class CompanyStep extends React.Component {
                         containerStyles={[styles.mt4]}
                         disabled={shouldDisableCompanyName}
                         defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'companyName')}
+                        shouldSaveDraft
                     />
                     <AddressForm
                         translate={this.props.translate}
```

Comment:
We need to pass default values to `AddressForm` and use it in `AddressForm` input, e.g.:
```suggestion
                        translate={this.props.translate}
                        defaultValues={{
                            street: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'addressStreet'),
                            city: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'addressCity'),
                            state: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'addressState'),
                            zipCode: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'addressZipCode'),
                        }}
```

And in `AddressForm`:

value={props.values.street || props.defaultValues.street}

Reply:
I have passed suggested default values to `AddressForm`.

Inside `AddressForm` I have added default values as `defaultValue=props.defaultValues.street}` because providing default value to `value` prop causing bug which prevent us from deleting anything from the field. I have tested it's working as expected.